tests/data-files/examples/example-info.csv

```diff
@@ -123,6 +123,16 @@
 "iserv-0.6.1/catalog.json","COLLECTION","0.6.1",""
 "hand-0.9.0/collection.json","COLLECTION","0.9.0",""
 "hand-0.8.1/collection.json","COLLECTION","0.8.1",""
+"1.0.0-RC1/extended-item.json","ITEM","1.0.0-rc.1","eo|view|projection|scientific"
+"1.0.0-RC2/extended-item.json","ITEM","1.0.0-rc.2","https://stac-extensions.github.io/remote-data/v1.0.0/schema.json|https://stac-extensions.github.io/view/v1.0.0/schema.json|https://stac-extensions.github.io/projection/v1.0.0/schema.json|https://stac-extensions.github.io/scientific/v1.0.0/schema.json|https://stac-extensions.github.io/eo/v1.0.0/schema.json"
+"1.0.0-RC3/catalog.json","CATALOG","1.0.0-rc.3",""
+"1.0.0-RC3/collection-only/collection.json","COLLECTION","1.0.0-rc.3","https://stac-extensions.github.io/eo/v1.0.0/schema.json|https://stac-extensions.github.io/projection/v1.0.0/schema.json|https://stac-extensions.github.io/view/v1.0.0/schema.json"
+"1.0.0-RC3/collection.json","COLLECTION","1.0.0-rc.3","https://stac-extensions.github.io/eo/v1.0.0/schema.json|https://stac-extensions.github.io/view/v1.0.0/schema.json"
+"1.0.0-RC3/collectionless-item.json","ITEM","1.0.0-rc.3","https://stac-extensions.github.io/eo/v1.0.0/schema.json|https://stac-extensions.github.io/view/v1.0.0/schema.json"
+"1.0.0-RC3/core-item.json","ITEM","1.0.0-rc.3",""
+"1.0.0-RC3/extended-item.json","ITEM","1.0.0-rc.3","https://stac-extensions.github.io/eo/v1.0.0/schema.json|https://stac-extensions.github.io/projection/v1.0.0/schema.json|https://stac-extensions.github.io/scientific/v1.0.0/schema.json|https://stac-extensions.github.io/view/v1.0.0/schema.json|https://stac-extensions.github.io/remote-data/v1.0.0/schema.json"
+"1.0.0-RC3/extensions-collection/collection.json","COLLECTION","1.0.0-rc.3",""
+"1.0.0-RC3/extensions-collection/proj-example/proj-example.json","ITEM","1.0.0-rc.3","https://stac-extensions.github.io/eo/v1.0.0/schema.json|https://stac-extensions.github.io/projection/v1.0.0/schema.json"
```

Comment:
Looks like these files don't exist yet, but once that's fixed this looks good to me, too.

Reply:
Actually, I think these just need to be changed to `1.0.0-RC2`. @jonhealy1 I'm happy to merge in `main` and update this if you want. I think this PR is ready to go other than this.

Reply:
I think I only added in one rc1 and one rc2 entry. I think the rc3 stuff comes from a previous iteration of what I was building off of? If you can get it working to merge that would be great. Cheers.

Reply:
Ah, yeah, I see that now. I'll try to work through that and get the tests working again.